actors/runtime/indices/indices.go

```diff
@@ -12,66 +10,22 @@ import (
 // it is a passive data structure that allows for convenience access to network indices
 // and pure functions in implementing economic policies given states
 type Indices interface {
-	Epoch() abi.ChainEpoch
-	NetworkKPI() big.Int
-	TotalNetworkSectorWeight() abi.SectorWeight
-	TotalPledgeCollateral() abi.TokenAmount
-	TotalNetworkEffectivePower() abi.StoragePower // power above minimum miner size
-	TotalNetworkPower() abi.StoragePower          // total network power irrespective of meeting minimum miner size
-
-	TotalMinedFIL() abi.TokenAmount
-	TotalUnminedFIL() abi.TokenAmount
-	TotalBurnedFIL() abi.TokenAmount
-	LastEpochReward() abi.TokenAmount
-
-	StorageDeal_DurationBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-	) (minDuration abi.ChainEpoch, maxDuration abi.ChainEpoch)
-	StorageDeal_StoragePricePerEpochBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-		endEpoch abi.ChainEpoch,
-	) (minPrice abi.TokenAmount, maxPrice abi.TokenAmount)
-	StorageDeal_ProviderCollateralBounds(
```

Comment:
Blocking: This function (and others) was previously a method on Indices, which enabled it to be time/KPI-dependent. Making them non-time/KPI-dependent is a cryptoecon policy change, which we would need to evaluate separately.

Reply:
When there is a concrete function that depends on some external quantity, we can restore it appropriately.